packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/IODetails/IODetails.tsx

```diff
@@ -65,10 +65,13 @@ const IOItem = ({ anonymitySet, address, symbol, amount, isPhishingTransaction }
                     {anonymity && <UtxoAnonymity anonymity={anonymity} />}
                     {amount &&
                         (typeof amount === 'string' && symbol ? (
-                            <FormattedCryptoAmount
-                                value={formatNetworkAmount(amount, symbol)}
-                                symbol={symbol}
-                            />
+                            <Column>
+                                <br />
```

Comment:
why not to use margin top or something else instead of `<br />`?

Reply:
Sure, used a margin